wagtailio/blog/templates/blog/blog_index_page.html

```diff
@@ -3,7 +3,6 @@
 {% load static wagtailcore_tags wagtailadmin_tags wagtailimages_tags wagtailio_utils %}
 
 {% block body_class %}template-blog{% endblock %}
-{% block extra_css %}<link rel="stylesheet" href="{% static 'css/vendor/code-highlighting.css' %}" />{% endblock %}
```

Comment:
My preferred solution would be to bundle up code-highlighting.css so it can be used on the blog.

Reply:
Can do. I'd assumed that the new design would bring with it new code block styles but perhaps it's better to re-use this.

Reply:
I've added them to vendor which will be bundled with the main css file